### DISCUSSION TITLE
[Bug]: asideMode="dismiss" компонента Banner

### DISCUSSION THREAD

**Discussion by revidovich** [VKCOM/VKUI#4142] [Q&A] [answered]
### Описание

кнопка крестика не закрывает баннер

### Версия

5.1.0

### Шаги воспроизведения

нажать на крестик компонента
<Banner
                mode="image"
                size="m"
                header="Где найти лучшие скидки?"
                subheader={
                    <span>
                        Билеты куда угодно
                    </span>
                }
                background={<div className={s.banner} />}
                asideMode="dismiss"
                actions={
                    <Button appearance="overlay" size="m" >
                        Подробнее
                    </Button>
                }
/>

### Ожидаемое поведение

кнопка крестика скрывает баннер

### Скриншоты

![image](https://user-images.githubusercontent.com/63742797/216777634-a1909917-556b-4933-849f-ba13149f52cd.png)

**Comment by SevereCloud** [comment, marked as answer]
Привет, для обработки нажатия на крестик используйте свойство [`onDismiss`](https://vkcom.github.io/VKUI/#/Banner:~:text=%D0%A2%D0%B8%D0%BF%20%D0%B1%D0%B0%D0%BD%D0%BD%D0%B5%D1%80%D0%B0.-,onDismiss,-MouseEventHandler%3CHTMLButtonElement%3E%20%7C%20undefined)

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/14944123/216782480-16186447-869a-445c-b852-98a91b4fc469.png">

[Пример закрытия](https://codesandbox.io/s/banner-ondismiss-qoll3r?file=/src/App.tsx)

**Comment by revidovich** [comment]
спасибо!
